client-src/elements/chromedash-feature-page_test.js

```diff
@@ -91,8 +91,8 @@ describe('chromedash-feature-page', () => {
         owners: ['fake chrome owner one', 'fake chrome owner two'],
         status: {text: 'fake chrome status text'},
       },
-      ff: {view: {text: 'fake ff view text'}},
-      safari: {view: {text: 'fake safari view text'}},
+      ff: {view: {val: 1, text: 'fake ff view text', url: 'fake ff url'}},
```

Comment:
The tests now render the case where val is truthy and will show the links. But now the tests don't assert the behavior of when the val is falsy.